### PR TITLE
Clean up for rubocop

### PR DESCRIPTION
**Pull request by silug**

### CODE REVIEW

File: lib/puppet/functions/simplib/ip/family_hash.rb [hunk 1]
```diff
@@ -80,8 +82,8 @@ def family_hash(addresses)
       begin
         ip = IPAddr.new(addr)
 
-        addr_normalized, cidr_netmask = call_function('simplib::nets2cidr', addr).
-          first.split('/')
+        addr_normalized, cidr_netmask = call_function('simplib::nets2cidr', addr)
+                                        .first.split('/')
```
Comment (rgardner4012) [inline lib/puppet/functions/simplib/ip/family_hash.rb:86]: Is this whitespace preferred?

Comment (silug) [reply lib/puppet/functions/simplib/ip/family_hash.rb:86]: > Is this whitespace preferred?

Yeah, that looks right.

File: spec/functions/module_metadata/os_supported_spec.rb [hunk 2]
```diff
@@ -14,87 +14,93 @@
...
-              'operatingsystem'        => 'Aardvark',
-              'operatingsystemrelease' => [ '14.999' ]
-            },
-            {
-              'operatingsystem'        => 'Banana',
-              'operatingsystemrelease' => [ '15.123' ]
-            }
-          ]
-        }}
+        let(:module_metadata) do
```
Comment (rgardner4012) [inline spec/functions/module_metadata/os_supported_spec.rb:46]: Is the indentation for name, and the spacing of the closing braces & brackets good here?

Comment (silug) [reply spec/functions/module_metadata/os_supported_spec.rb:46]: > Is the indentation for name, and the spacing of the closing braces & brackets good here?

Nope, I missed that.  Thanks.

**Review by rgardner4012 — APPROVED** [review]

**Pull request merged**